test/extended/testdata/build-secrets/s2i-binary-dir/.s2i/bin/assemble

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copy secrets into a location they can be output during image run
+mkdir -p $HOME/testsecret
+[[ -f /tmp/secret1 ]] && cp /tmp/secret? $HOME/testsecret 
```

Comment:
There are actually three files /tmp/secret{1|2|3}, so the wildcard is required . Though I'm not certain why the original author was even checking for the existence of a file before the copy operation. In any case, the updated scripts actually use the outcome of the test.